src/server/responses.py

```diff
@@ -80,3 +82,13 @@ def internal_database_error(_message: Message) -> Response:
             {"message": "Internal server error"},
             500,
         )
+
+    @staticmethod
+    def chart(filtered_list: list[dict]) -> Response:
+        """200: success on chart endpoint."""
+        return make_response(filtered_list)
+
+    @staticmethod
+    def chart_missing_parameters_error() -> Response:
+        """400: missing parameters in chart endpoint."""
+        return make_response({"message": "Please provide both from and to parameters!"}, 400)
```

Comment:
Right now respones are sorted by response number, would be nice to move them respectively. Also expand make_response to multiple lines like the other functions, so that the formatting is consistent.

Reply:
In my opinion it makes more sense to keep chart endpoint related responses together instead of mixing them with other unrelated endpoints just to keep response numbers sorted.

Reply:
Sure, maybe we'll work out a better way to group these along the way.